File: backend/src/main/java/com/ssafy/db/entity/User/UserProfile.java
package com.ssafy.db.entity.User;


import com.ssafy.db.entity.Guild.Guild;
import com.ssafy.db.entity.Pet.Pet;
import lombok.Getter;
import lombok.Setter;
import org.hibernate.annotations.ColumnDefault;

import javax.persistence.*;

@Getter
@Setter
@Entity
public class UserProfile {

    @Id
    @Column(name = "user_nickname")
    private String nickname;

    /* About Todo/Quest Point */
    @ColumnDefault("0")
    private int point;
    @ColumnDefault("3")
    private int rest_point;

    // 펫 Item
    private int hat;
    private int dress;
    private int background;

    // 위도
    private double latitude;
    // 경도
    private double longitude;
    // 위치 이름 저장
    private String location;
    // 완료해야하는 시간
    private int finish_datetime;

    // 유저 만보기
    private int pedometer;

}

File: mobile/app/src/main/java/com/team/teamrestructuring/dto/UserProfile.kt
package com.team.teamrestructuring.dto

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class UserProfile(
    @Expose
    @SerializedName("nickname")
    var nickname:String,

    @Expose
    @SerializedName("point")
    var point:Int,

    @Expose
    @SerializedName("rest_point")
    var rest_point:Int,

    @Expose
    @SerializedName("finish_datetime")
    var time : Int = -1,

    @Expose
    @SerializedName("location")
    var location:String="",

    @Expose
    @SerializedName("latitude")
    var latitude:Double = 0.0,

    @Expose
    @SerializedName("longitude")
    var longitude:Double = 0.0


){


    override fun toString(): String {
        return "UserProfile(nickname='$nickname', point=$point, rest_point=$rest_point, time=$time, location='$location', latitude=$latitude, longitude=$longitude)"
    }
}